src/main/telemetry/smartport.c

```diff
@@ -42,6 +42,7 @@
 #include "sensors/battery.h"
 #include "sensors/acceleration.h"
 #include "sensors/barometer.h"
+#include "sensors/pitotmeter.h"
```

Comment:
OK, got it. However in this case `smartport.c` itself should not include `config_master.h`. Similarly for `ltm.c`.

This may seem like nitpicking, but the build time is already too long and will only get longer as new targets are added. Guarding against unnecessary `#includes` helps reduce the build time.